server/controllers/routes/post/getDraftPosts.js

```diff
@@ -3,17 +3,17 @@ const { getEvents, getPublicServices } = require('../../../database/queries/getP
 const draftPosts = async (req, res, next) => {
   try {
     const { id: publisherId } = req.user;
-    const resEvent = await getEvents('true', publisherId)
-    const resPublic = await getPublicServices('true', publisherId)
-     
+    const resEvent = await getEvents('true', publisherId);
+    const resPublic = await getPublicServices('true', publisherId);
+    resEvent.rows.forEach(event => event.type = 'event');
```

Comment:
i think instead of iterate on all element you can determine the type in the response link 
```js
 res.status(200).send({
      data: { events: resEvent, publicServices: resPublic },statusCode: 200,
    })	 ;
```